visualization/src/resource-visualization.cc

```diff
@@ -562,66 +562,85 @@ void visualizeReprojectedDepthResourcePerRobot(
       !(FLAGS_vis_pointcloud_accumulated_before_publishing &&
         FLAGS_vis_pointcloud_publish_in_sensor_frame_with_tf));
 
-  // Accumulate all the point clouds of the robot missoins. If there are no
-  // missions in the map, a empty point cloud will be published.
-  for (const auto& kv : robot_name_to_mission_ids_map) {
-    const std::string& robot_name = kv.first;
-    const vi_map::MissionIdList& mission_ids = kv.second;
-
-    if (robot_name.empty()) {
-      LOG(ERROR) << "Cannot visualize point clouds for robot missions, because "
-                 << "the robot name is empty (name: '" << robot_name << "')";
-      continue;
-    }
-
-    resources::PointCloud accumulated_point_cloud_G;
-    for (const vi_map::MissionId& mission_id : mission_ids) {
-      if (!mission_id.isValid()) {
-        LOG(ERROR) << "Cannot visualize one mission of robot '" << robot_name
-                   << "' since the provided mission id is invalid!'";
-        continue;
-      }
+  // Accumulate all the point clouds of the robot missions. If there are no
+  // missions in the map, an empty point cloud will be published.
+  std::function<void(const std::vector<size_t>&)>
```

Comment:
this formatting looks a bit strange?

Reply:
indeed, the linter did it automatically though